lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/environment/LidarUpdates.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.interactions.environment;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+import org.eclipse.mosaic.lib.spatial.PointCloud;
+import org.eclipse.mosaic.rti.api.Interaction;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LidarUpdates extends Interaction {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * String identifying the type of this interaction.
+     */
+    public final static String TYPE_ID = createTypeIdentifier(LidarUpdates.class);
+
+    /**
+     * Time at which the next sensor update will be sent.
+     */
+    private long nextUpdate;
+
+
+    public record LidarUpdate(String unitId, PointCloud pointCloud) {}
```

Comment:
What do we think about this style-wise @kschrab? Typically, we placed in-line classes at the bottom of a class what should we do with records?

Reply:
>  Typically, we placed in-line classes at the bottom of a class

do we? 🤔

Reply:
At least that's what I did. I think at the top of the class makes a lot of sense too. Just not somewhere in between other declarations.